README.md

```diff
@@ -27,8 +27,8 @@ SAM has only one hyperparameter namely `rho` that controls the neighborhood of t
 | Without SAM |            0.575114            |           82.9          |
```

Comment:
Did you also want to update this table with the final scores?

Reply:
Yes It was updated in the previous commit. ;)

Reply:
Oh I see. Anyway the current README should reflect the changes you made. Thank you so much.